packages/compass-import-export/package.json

```diff
@@ -1,171 +1,122 @@
 {
   "name": "@mongodb-js/compass-import-export",
-  "productName": "Compass Import/Export",
-  "version": "6.21.0",
-  "apiVersion": "3.0.0",
-  "description": "Compass Import/Export Plugin",
-  "homepage": "https://github.com/mongodb-js/compass",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mongodb-js/compass.git"
+  "description": "Compass Import/Export",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
   },
   "bugs": {
     "url": "https://jira.mongodb.org/projects/COMPASS/issues",
     "email": "compass@mongodb.com"
   },
-  "main": "lib/index.js",
+  "homepage": "https://github.com/mongodb-js/compass",
+  "version": "6.21.0",
+  "productName": "Compass Import/Export",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/compass.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "license": "SSPL",
+  "main": "dist/index.js",
   "exports": {
-    "webpack": "./src/index.js",
-    "require": "./lib/index.js"
+    "webpack": "./src/index.ts",
+    "require": "./dist/index.js"
   },
+  "types": "./dist/index.d.ts",
   "scripts": {
-    "clean": "rimraf lib",
-    "precompile": "npm run clean",
-    "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js",
-    "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
-    "pretest": "mongodb-runner start --port=27018",
-    "posttest": "mongodb-runner stop --port=27018",
-    "test:watch": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" --watch",
-    "test:dev": "cross-env NODE_ENV=test mocha-webpack",
-    "cover": "nyc npm run test",
-    "check": "npm run lint && npm run depcheck",
     "prepublishOnly": "npm run compile",
-    "lint": "eslint \"./src/**/*.{js,jsx}\" \"./test/**/*.js\"",
+    "compile": "npm run webpack -- --mode production",
+    "prewebpack": "rimraf ./lib",
+    "webpack": "webpack-compass",
+    "start": "npm run webpack serve -- --mode development",
+    "analyze": "npm run webpack -- --mode production --analyze",
+    "typecheck": "tsc --noEmit",
+    "eslint": "eslint",
+    "prettier": "prettier",
+    "lint": "npm run eslint . && npm run prettier -- --check .",
     "depcheck": "depcheck",
-    "test-ci": "npm run test",
-    "bootstrap": "npm run compile"
+    "check": "npm run typecheck && npm run lint && npm run depcheck",
+    "check-ci": "npm run check",
+    "pretest": "mongodb-runner start --port=27018",
```

Comment:
You need to ass the same pre-post script for test-electron command, because it's not there CI is failing currently